.github/workflows/README.md

```diff
@@ -1,38 +1,43 @@
-# How to use those workflows
+Here is the updated documentation:
+
+# How to use these workflows
 
 There are a few [GitHub secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) to configure to fully leverage the build.
 
-You can use and set the followings secrets also in your fork.
+You can use and set the following secrets also in your fork.
+
+## Tunnelmole and Ngrok Debugging
 
-## Ngrok Debugging
+You can debug a GitHub Action build using [Tunnelmole](https://tunnelmole.com) or [NGROK](https://ngrok.com/). Tunnelmole is a free and open source tunneling tool, while NGROK is a popular closed source tunneling tool.
 
-You can debug a GitHub Action build using [NGROK](https://ngrok.com/).
+Debugging is disabled for automated builds triggered by push and pull_requests.
 
-It is disabled for automated build triggered by push and pull_requests.
+You can trigger a workflow run manually to enable debugging.
 
-You can trigger a workflow run manually  enabling ngrok debugging.
+It will open an ssh connection to the VM and keep it up and running for one hour. The connection URL is shown in the log for `debugAction.sh`. 
 
-It will open an ssh connection to the VM and keep it up and running for one hour.
-The connection url is showns in the log for debugAction.sh
+You can then connect to the build VM, and debug it. You need to use a password of your choice to access it.
 
-You can then connect to the build vm, and debug it.
-You need to use a password of your choice to access it.
+You can continue the build with `touch /tmp/continue`. You can abort the build with `touch /tmp/abort`.
 
-You can continue the build with `touch /tmp/continue`.
-You can abort the build with `touch /tmp/abort`.
+### Tunnelmole
```

Comment:
Thank you for your contribution!

I am not familiar with Tunnelmole.
Our on-demand workflow is based on [ngrok setup](https://github.com/apache/openwhisk/blob/master/tools/github/debugAction.sh#L48).
Don't we need a similar change in the workflow file?